Makefile

```diff
@@ -43,7 +43,8 @@ down:
 
 vas:
 	git clone https://github.com/intel/video-analytics-serving && \
-	cd video-analytics-serving && \
+	cd video-analytics-serving/docker && \
```

Comment:
Is there an easy way to make this configurable?

Reply:
Made configurable